AWSWindowsHelpers/public/ARN/Get-AWSWindowsHelpersARNDetail.ps1

```diff
@@ -0,0 +1,128 @@
+<#
+.Synopsis
+   Returns the details of an AWS Resource from the ARN
+.DESCRIPTION
+   Returns the details of an AWS Resource from the ARN
+.EXAMPLE
+   Get-AWSWindowsHelpersARNDetail -arn "arn:aws:iam::123456789012:server-certificate/2017_wild_example_com"
+.INPUTS
+   arn - ARN of the AWS certificate
+.OUTPUTS
+   Returns a object with the following properties
+   Partition - The partition the resource is in
+   Service - The AWS Service
+   Region - The AWS region
+   AccountID - AWS account number
+   ResourceDetail - The resource ARN detail
+   ResourceType - The type of AWS resource
+.FUNCTIONALITY
+   Retrieves from a ARN the details of the AWS service
+#>
+Function Get-AWSWindowsHelpersARNDetail
+{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$true)]
+        [ValidatePattern('^arn:*.:*.:*.:*.:*.')]
```

Comment:
is `^arn:*.:*.:*.:*.:*.` right? or is the final `*.` a transposed `.*`?

Reply:
You're right, it should be `.*` and not `*.` for those elements.